src/libopensc/iso7816.c

```diff
@@ -391,6 +391,8 @@ iso7816_process_fci(struct sc_card *card, struct sc_file *file,
 			}
 			sc_log(ctx, "  type: %s", type);
 			sc_log(ctx, "  EF structure: %d", byte & 0x07);
+			sc_log(ctx, "  tag 0x82: 0x%02x", byte);
+			sc_file_set_type_attr(file, &byte, 1);
```

Comment:
tag 82 may be followed by a variable size of information between one and 6 bytes. You should instead use
```
sc_file_set_type_attr(file, tag, taglen);
```